test/Dialect/ESI/services.mlir

```diff
@@ -0,0 +1,26 @@
+// RUN: circt-opt %s | circt-opt | FileCheck %s
```

Comment:
Suggesting renaming this file to `services_round_trip.mlir` to be consistent with naming used in other dialect tests.